src/graphics/Graphviz.jl

```diff
@@ -275,4 +275,18 @@ end
 
 indent(io::IO, n::Int) = print(io, " "^n)
 
+function to_graphviz_with_viewer(g::Graph; path::String="", kw...)
+    filepath = isempty(path) ? "$(tempname()).png" : path
+    open(filepath, "w") do io
+        run_graphviz(io, to_graphviz(g), format="png")
+    end
+    if Sys.islinux()
+        run(`feh $filepath`, wait=false)
```

Comment:
Perhaps there is no standard way to do this, but on my system it's `xdg-open`. I don't have `feh`.